package.json

```diff
@@ -12,6 +12,7 @@
     "lint": "eslint packages/ scripts/ --ext=js,ts",
     "format": "prettier --write '{packages,scripts}/**/*.{js,ts,json,md}'",
     "build": "lerna run build --ignore perf-benchmarks --ignore integration-tests",
+    "bundlesize": "bundlesize --config bundlesize.config.json",
```

Comment:
I would also recommend adding this as a `precommit` hook. It is quite frustrating to wait a couple of minutes before the CI bails when such an inexpensive check can also be done locally.

Reply:
Subjective nitpick, I personally prefer not to pollute the root directory with all the config files. 
I would have put it under `scripts/bundlesize`.

Reply:
For the `precommit` hook, wouldn't that assume we've gone through the full build process (which takes almost 50s on my machine)? Personally I would prefer to have a slightly faster commit process, even if it means I find out about a failure later in CI.

Reply:
Would we want a pre-push hook then instead?  That way we can check locally before it reaches the CI and wouldn't affect the commits.

Reply:
Personally I'd rather just have it in CI but I'm happy to take the question to the group to see what they think.

Reply:
Sounds good, I'll ask.

Reply:
How long does the check take?

Reply:
When I run the bundle size check locally it takes 0.53 seconds and it takes 29.43 seconds for yarn build to complete.

Reply:
To test locally I think we would want a new script that runs something like:

yarn build && yarn bundlesize

Reply:
After discussing on slack, I'll keep this check in CI.  The time to run bundlesize is quick but the time to build is much longer and may vary depending on the machine.